doc/reference/filters.md

```diff
@@ -947,6 +950,73 @@ filters:
 | ...                                                                         |
 | result9                                                                     |
 
+## DataBuilder
+
+DataBuilder is used to manipulate and store data. The data from the previous
+filter can be transformed and stored in the context so that the data can be
+used in subsequent filters.
+
+For example, we can use DataBuilder to store token from login request, and
```

Comment:
the example is correct, but the issue is that the request is never sent, please refer to other filters like `RequestBuilder` or `ResponseBuilder`.

btw, a completed example is not a must, it is fine to just show how to use it and how to use the data it generates.

Reply:
Changed the sample to a simpler one.